### PR TITLE
1479 banked months evaluator

### PR DESCRIPTION
**Pull request by IlseFerris**

### CODE REVIEW

File: actions/banked-months-updater.vbs [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+		cancel_without_confirmation
+		Call validate_MAXIS_case_number(err_msg, "*")
+		Call validate_footer_month_entry(initial_month, initial_year, err_msg, "*")
+		If trim(worker_signature) = "" then err_msg = err_msg & vbNewLine & "* Sign your case note."
+		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
+	LOOP UNTIL err_msg = ""
+call check_for_password(are_we_passworded_out)
+Loop until are_we_passworded_out = false
+
+Call_check_for_MAXIS(False)
```
Comment (MDR22) [inline actions/banked-months-updater.vbs:86]: Extra underscore after Call causes script to error out.

File: actions/banked-months-updater.vbs [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+
+Call navigate_to_MAXIS_screen_review_PRIV("CASE", "CURR", is_this_priv)
+If is_this_priv = True then script_end_procedure("This case is privilged, and you do not have access. The script will now end.")
```
Comment (MDR22) [inline actions/banked-months-updater.vbs:89]: Typo - privileged

File: actions/banked-months-updater.vbs [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+    Call write_variable_in_case_note("--")
+	stats_counter = STATS_counter + 1
+Next 
+If update_wreg = True then Call write_variable_in_case_note("* The STAT/WREG panel was updated through CM + 1 for members who have banked months available, or who have exhausted them.")
+If spec_memo = True then Call write_variable_in_case_note("* A SPEC/MEMO was sent to the household with banked months and time-limited SNAP information.")
+Call write_variable_in_case_note("---")
+CALL write_variable_in_CASE_NOTE(worker_signature)
+stats_counter = STATS_counter + 1
+
+end_msg = "The main EATS household has been assessed and updated for banked months. Make sure to APP new SNAP results, and use NOTES - ELGIBILTY SUMMARY to case note the eligibility."
```
Comment (MDR22) [inline actions/banked-months-updater.vbs:435]: Small typo (ELGIBILTY -> ELIGIBILITY)

File: actions/banked-months-updater.vbs [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+BeginDialog Dialog1, 0, 0, 550, 385, "Banked Months Evaluation Dialog for ABAWD/TLR's on Case #" & MAXIS_case_number
+  Text 65, 10, 455, 10, "**The following information are for the ABAWD and Banked Months counts/months members in the EATS Household containing Memb 01**"
+  Text 115, 25, 315, 10, "--The information below reflects counts from the initial month/year selected in the initial dialog.--"
+
+For i = 0 to Ubound(banked_months_array, 2)
+	If banked_months_array(abawd_status_const, i) = "10" or banked_months_array(abawd_status_const, i) = "13" then 
+		y_pos = (45 + i * 50)
+		GroupBox 10, y_pos, 530, 45, "MEMB #" & banked_months_array(member_number_const , i) & " " & banked_months_array(member_first_name_const, i) & ", ABAWD/FSET: " & banked_months_array(wreg_status_const, i) & "/" & banked_months_array(abawd_status_const, i)
+  		Text 20, y_pos + 15, 255, 10, "ABAWD Count/Months Used: " & banked_months_array(abawd_mo_count_const, i) & " - " & banked_months_array(abawd_mo_string_const, i)
+  		Text 20, y_pos + 30, 255, 10, "ABAWD Months Evaluation: " & banked_months_array(abawd_month_eval_const, i)
```
Comment (MDR22) [inline actions/banked-months-updater.vbs:325]: Depending on the length of banked_months_array(abawd_month_eval_const, i) I found that the group box cut off the text. I can provide a screenshot if that is helpful. May want to increase the size of the group text box?

**Review by MDR22 — CHANGES_REQUESTED** [review]
I tested the script on a few cases in training and it seemed to work as expected. I noted an issue with a function call and a few typos, along with a dialog formatting consideration.

**Comment by MeganGeissler** [comment]
I didn't find anything outside of what Mark found! :)

**Review by MeganGeissler — APPROVED** [review]

File: actions/banked-months-updater.vbs (hunk 1 above)
Comment (IlseFerris) [reply actions/banked-months-updater.vbs:86]: Oh my gosh, I've done this SO much on this script! Never have I ever had this many problems with it, ha!

File: actions/banked-months-updater.vbs (hunk 3 above)
Comment (IlseFerris) [reply actions/banked-months-updater.vbs:435]: Do you like how I spelled it wrong and right in the same sentence @MDR22?

**Pull request merged**